odoo/addons/openupgrade_records/models/analysis_wizard.py

```diff
@@ -64,6 +64,7 @@ def write_file(module, version, contents,
         res = compare.compare_sets(remote_records, local_records)
 
         # Retrieve xml id representations and compare
+        # In version 11, add 'noupdate' as a field to retrieve and compare
```

Comment:
Why this comment here? Is it a TODO?

Reply:
Yes, it is a TODO for when this is ported to OpenUpgrade 11. If we add the field now, it will break the analysis between 9 and 10 because 9 does not have this code.

Reply:
Then maybe you can add the `TODO` or `FIXME` word before.

Reply:
Oh common @pedrobaeza.

Reply:
OK, let's keep it as is, but if we won't remember to change this on v11, it's not my fault :stuck_out_tongue_winking_eye: